src/client/containers/Profile/index.js

```diff
@@ -1,16 +1,28 @@
-/* eslint-disable react/jsx-props-no-spreading */
-import React from 'react';
+import React, { useContext } from 'react';
 import ProfileDetails from '../../components/Profile/ProfileDetails';
 import { UserContext } from '../../context/userContext';
 
 export default function Profile() {
+  const user = useContext(UserContext);
+  const userUid = user && user.uid;
+
+  const handleSubmit = async (formDetails) => {
+    const getUser = await fetch(`/api/users/current?uid=${userUid}`);
```

Comment:
Remember to import the function aswell. When you use this function you dont need to say .json

```suggestion
    const getUser = await fetchWithAuth(`/api/users/current?uid=${userUid}`);
```